src/components/CreateVmWizard/steps/BasicSettings.js

```diff
@@ -198,6 +199,26 @@ class BasicSettings extends React.Component {
       (checkInit ? okInitHostname : true)
   }
 
+  /**
+   * supports in showing high performance option for HP template based vm.
+   * the high performance option won't appear in any other case.
+   **/
+  buildOptimizedForList (dataSet) {
+    const optimizedForList = { ...optimizedForMap }
+
+    // TODO: remove when web-ui fully supports creating high performance VM
```

Comment:
please change the TODO comment position to relate to the whole method and also rephrase to something like "remove when REST API will fully support creating of a High Performance VM type, not based on a template"